examples/zh_claim_level_example.ipynb

```diff
@@ -0,0 +1,520 @@
+       "  2.765898e-05,\n",
+       "  2.765898e-05,\n",
+       "  2.765898e-05,\n",
+       "  2.765898e-05,\n",
+       "  1.8441091e-05]]"
+      ]
+     },
+     "execution_count": 25,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Maximum Token Entropy\n",
+    "max_token_ent = MaxTokenEntropyClaim()\n",
+    "max_token_ent(stat)  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "2f118841-c40b-4d04-b06f-da7a6b1c287c",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[[-72.85801413972513,\n",
+       "  -30.04430177868926,\n",
+       "  -47.79385693371296,\n",
+       "  -26.227787380106747,\n",
+       "  -107.58818900730694,\n",
+       "  -107.45839975989657,\n",
+       "  -108.33176562050357,\n",
+       "  -106.02369641518453,\n",
+       "  -128.93925437084226]]"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Pointwise Mutual Information\n",
+    "pmi = PointwiseMutualInformationClaim()\n",
+    "pmi (stat)  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ef7ffc38-e905-42eb-aa48-5dee774afed1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# PTrue\n",
+    "ptrue_claim = PTrueClaim()\n",
+    "ptrue_claim (stat)  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "42d9982a",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Some weights of the model checkpoint at microsoft/deberta-large-mnli were not used when initializing DebertaForSequenceClassification: ['config']\n",
+      "- This IS expected if you are initializing DebertaForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
+      "- This IS NOT expected if you are initializing DebertaForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[[-0.7113813961415233,\n",
+       "  -0.8446114377545357,\n",
+       "  -0.2803199309110498,\n",
+       "  -0.7791504012475273,\n",
+       "  -0.5517237721010816,\n",
+       "  -0.557831080274213,\n",
+       "  -0.5316904995544154,\n",
+       "  -0.5553218365419984,\n",
+       "  -0.7292863406843685]]"
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Claim Conditional Probability\n",
+    "for calculator in [\n",
+    "    GreedyAlternativesNLICalculator(Deberta())\n",
```

Comment:
The default Deberta path `microsoft/deberta-large-mnli` is only suitable for English texts. 
I propose specifying here `Deberta(deberta_path="MoritzLaurer/mDeBERTa-v3-base-xnli-multilingual-nli-2mil7")`. This model is specifically trained on multilingual NLI tasks and should provide better performance for Chinese inputs.

Reply:
Thanks for telling me, already replaced.